src/Compilers/CSharp/Test/Emit3/ExtensionTypeTests.cs

```diff
@@ -140,6 +140,41 @@ static void checkBaseExtension(NamedTypeSymbol baseExtension)
         }
     }
 
+    [Theory, CombinatorialData]
+    public void TestCSharp12(bool isExplicit)
+    {
+        var src = $$"""
+            {{(isExplicit ? "explicit" : "implicit")}} extension E { }
+            """;
+        var comp = CreateCompilation(src, parseOptions: TestOptions.Regular12, targetFramework: TargetFramework.Net70)
+            .VerifyEmitDiagnostics(
```

Comment:
>VerifyEmitDiagnostics [](http://example.com/codeflow?start=13&length=21)

For language version testing we usually trying to cover three scenarios:
- Previous version, expected to complain about the version
- TestOptions.RegularNext - expected to not complain
- TestOptions.RegularPreview - expected to not complain

Reply:
done.